tests/web/html/all-tests.html

```diff
@@ -11,25 +11,25 @@
     <script src="../../../node_modules/infusion/tests/lib/qunit/addons/composite/qunit-composite.js"></script>
     <script>
     QUnit.testSuites([
-        "../../../gpii/node_modules/matchMakerFramework/test/html/MatchMakerUtilitiesTest.html",
-        "../../../gpii/node_modules/matchMakerFramework/test/html/InverseCapabilitiesTest.html",
-        "../../../gpii/node_modules/transformer/test/html/TransformerTests.html",
-        "../../../gpii/node_modules/journal/test/html/JournalIdParserTests.html",
-        "../../../gpii/node_modules/lifecycleManager/test/html/LifecycleManagerTest.html",
-        "../../../gpii/node_modules/lifecycleManager/test/html/DynamicComponentIndexerTest.html",
-        "../../../gpii/node_modules/ontologyHandler/test/html/OntologyHandlerUtilitiesTest.html",
-        "../../../gpii/node_modules/settingsHandlers/test/web/html/SettingsHandlerUtilitiesTest.html",
-        "../../../gpii/node_modules/contextManager/test/html/ContextManagerUtilitiesTests.html",
-        "../../../gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/test/html/AuthGrantFinderTests.html",
-        "../../../gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/test/html/AuthorizationServiceTests.html",
-        "../../../gpii/node_modules/gpii-oauth2/gpii-oauth2-datastore/test/html/DataSource-PouchDBTests.html",
-        "../../../gpii/node_modules/gpii-oauth2/gpii-oauth2-utilities/test/html/OAuth2UtilitiesTests.html",
-        "../../../gpii/node_modules/processReporter/test/web/html/ProcessesBridgeTest.html",
-        "../../../gpii/node_modules/processReporter/test/web/html/ProcessReporterTest.html"
+        "/gpii/node_modules/matchMakerFramework/test/html/MatchMakerUtilitiesTest.html",
```

Comment:
This change means that I can no longer run the web-based tests from my local HTTP server - is there any way we can do without it?

Reply:
I haven't found a way yet, no.  The parent directory calls end up bypassing the routing that loads the instrumented code.  You can run the tests from your local HTTP server as long as you host universal as the root.

Reply:
I can make a separate fixture for testem if you like.

Reply:
I don't suppose there's a way to detect whether we are running in testem or not, and issue the appropriate form of path?

Reply:
We have also discussed detecting the presence of the global Testem object and using that to choose which paths to use.